File: Examples/RealtorApp/DDD.Light.Realtor.REST.API/App_Start/AttributeRoutingHttpConfig.vb
Imports System.Web.Http
Imports AttributeRouting.Web.Http.WebHost

<assembly: WebActivator.PreApplicationStartMethod(GetType(DDD.Light.Realtor.REST.API.AttributeRoutingHttpConfig), "Start")>

Namespace DDD.Light.Realtor.REST.API
    Public Class AttributeRoutingHttpConfig
		Public Shared Sub RegisterRoutes(routes As HttpRouteCollection)
            
			' See http://github.com/mccalltd/AttributeRouting/wiki for more options.
			' To debug routes locally using the built in ASP.NET development server, go to /routes.axd
            
            routes.MapHttpAttributeRoutes()
		End Sub

        Public Shared Sub Start()
            RegisterRoutes(GlobalConfiguration.Configuration.Routes)
        End Sub
    End Class
End Namespace

File: Examples/RealtorApp/DDD.Light.Realtor.REST.API/App_Start/AttributeRoutingHttpConfig.cs
using System.Web.Http;
using AttributeRouting.Web.Http.WebHost;

[assembly: WebActivator.PreApplicationStartMethod(typeof(DDD.Light.Realtor.REST.API.AttributeRoutingHttpConfig), "Start")]

namespace DDD.Light.Realtor.REST.API 
{
    public static class AttributeRoutingHttpConfig
	{
		public static void RegisterRoutes(HttpRouteCollection routes) 
		{    
			// See http://github.com/mccalltd/AttributeRouting/wiki for more options.
			// To debug routes locally using the built in ASP.NET development server, go to /routes.axd

            routes.MapHttpAttributeRoutes();
		}

        public static void Start() 
		{
            RegisterRoutes(GlobalConfiguration.Configuration.Routes);
        }
    }
}
